### ISSUE TITLE
Time Zone Troubles                                                                                         Cycle:6(Heirlooms)     ( Day:6/23/20 (i believe it is day 2??)

### ISSUE THREAD

**Issue by KingCrabster** [jeanropke/RDR2CollectorsMap#1025]
#### Found a new location? [Use this form here](https://github.com/jeanropke/RDR2CollectorsMap/issues/182)
* What day cycle it is?
Cycle 6
* Coordinates or a clear image of the location
![jeanthing](https://user-images.githubusercontent.com/67336610/85436281-a4c18580-b53d-11ea-9431-c37832573d7b.png)

The Picture was taken at 8:30pm GMT+3,for some reason,it says there is supposed to be 20 hours and 30min until reset happens,
this is untrue since its supposed to reset at 12am GMT,but instead resets at 6pm GMT+3 (3pm GMT) 

Basically,when the reset  all the items on the map are wrong,since it thinks its the next day and shows the wrong locations

I am using:
Chrome  83.0.4103.116 64Bit
Windows 10 

* Include the item name
All items

Coordinates can be found by setting the `Show coordinates on click` to `Yes`
All of the Locations

**Comment by MichalD96** [comment]
I will look at it, the problem is i can't reproduce this bug in my browser, here everything works. For now you can enable debug features in menu, scroll to the bottom of menu, click "Enable day cycle changer" and on top of the menu you have buttons to change to the previous day.

**Comment by MichalD96** [comment]
Could you check if this bug also exists in other browsers like mozilla firefox?
could you paste this in to the console and make a screenshot of the result
`now = new Date();
new Date(Date.UTC(
      now.getUTCFullYear(),
      now.getUTCMonth(),
      now.getUTCDate()
    ));`

**Comment by KingCrabster** [comment]
Hey!First,yes the bug does in fact still happen in firefox,and second,this is what it shoed me in console 
![jeanthing2](https://user-images.githubusercontent.com/67336610/85520444-c95a4380-b5b7-11ea-9774-dfe1bb0e9b2c.png)

As you can see it says that it is Tuesday June 23 2020 %-m.but in GMT+3 it is actually 11:15am,June 24th

Hope this helps!

**Comment by KingCrabster** [comment]
This is weird,for some reason github is saying this was edited "10hours ago" i edited this just now,11:17am,you think this might be a github issue?

**Comment by MichalD96** [comment]
You have set wrong timezone on your PC, you are GMT+3, but your PC says GMT-7, and thats why you see this 10 hours difference.
Click start, type 'timezone', chose option "set time and timezone" and set time and timezone to automatic.

**Comment by KingCrabster** [comment]
Hey!i did what you said and put it on automatic,but it still shows that it will reset in 20 hours 30 minutes (currently 8:25pm GMT+3)

I used the debug menu to change the date and went to one of the spawns and it did indeed spawn in the correct item,so i guess its probably client side.

**Comment by MichalD96** [comment]
You can try to set your timezone manually in the same menu if automatic doesn't work, but i am sure there is problem.

**Comment by MichalD96** [comment]
Do you use VPN or proxy? It may cause that issues

**Comment by KingCrabster** [comment]
I do have a VPN(Nord) installed but i do not have it on currently,ill try to fix the timzone issue if i can
